crates/aptos-faucet/Cargo.toml

```diff
@@ -24,7 +24,7 @@ tokio = { version = "1.8.1", features = ["full"] }
 url = "2.2.2"
 warp = "0.3.2"
 
-generate-key = { path = "../../config/generate-key" }
+aptos = { path = "../aptos" }
```

Comment:
A bit late to the party but naming the crate `aptos` seems a bit vague to me. If this is tooling, while not `aptos-tool` or `aptos-cli`? Or is this also going to include compilable library code, e.g., to aid Rust development for Aptos services and apps?

Reply:
We can change this later.